src/Client.js

```diff
@@ -809,8 +809,8 @@ class Client extends EventEmitter {
                 return true;
             }
             const MAX_PIN_COUNT = 3;
-            if (window.Store.Chat.models.length > MAX_PIN_COUNT) {
-                let maxPinned = window.Store.Chat.models[MAX_PIN_COUNT - 1].pin;
+            if (window.Store.Chat._models.length > MAX_PIN_COUNT) {
```

Comment:
what about that?
`
window.Store.Chat.models.length ? window.Store.Chat.models.length : window.Store.Chat._models.length
`